docs/user-guide.md

```diff
@@ -139,3 +142,42 @@ kubectl create -f examples/volume/local-pv.yaml
 ```
 kubectl create -f examples/volume/pod-local.yaml
 ```
+
+#### NFS volume
+* create the NFS server
```

Comment:
can we add a link to `nfs-server.yaml` in k8s-shim project?

Reply:
updated

Reply:
updated